modules/util/css-builder/src/main/java/com/liferay/css/builder/maven/BuildCSSMojo.java

```diff
@@ -71,7 +71,7 @@ public void setDirNames(String dirNames) {
 	}
 
 	/**
-	 * @parameter
+	 * @parameter default-value="${project.build.directory}/${project.build.finalName}"
```

Comment:
you can for themes but not for portlets... I made a choice to add defaults for themes since all themes use the css builder but portlets don't always use it.